.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,9 +8,14 @@
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Test case or documentation update (no impact on source code)
+- [ ] Test automation update Unit or/and Functional (no changes in operator code)
+- [ ] Product update
+
+<!-- Different check list per type of change? -->
 
 ## Checklist
```

Comment:
do you think it will be useful to have a checklist for reviewers too? It's helpful to remind reviewers the things they should go through when reviewing a PR, and also record what they have done.